### PR TITLE
Add docs about asm.js/WASM/native differences 

### LINKED ISSUE #201: Add more information about asm.js/WASM/native differences
It would be great to have these (maybe reorganized a bit) added to the README:

> When running in the browser, the main difference is that WASM files need to be hosted and fetched, before initializing. Where asm.js can be bundled together with the lib. And then of course WASM should be faster.
>
> When running on the Node.js server, native should be faster. However there are Node.js environments where native modules are not supported (e.g. StackBlitz's WebContainers), or other JS runtimes that support WASM (e.g. Vercel's Edge Runtime, CloudFlare's worker, or Deno).
>
> Of course there are other difference between asm.js, native and WASM, such as security and compatibility.
>
> Overall there are many trade-offs between each choice, and it's better to pick the one that works the best for your use case.

_Originally posted by @shuding in https://github.com/vercel/satori/discussions/192#discussioncomment-3869126_

Thanks @wong2 for calling it out!

### PR DESCRIPTION
**Pull request by mrkldshv**
Closes #201.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -335,6 +335,14 @@ init(yoga)
 await satori(...)
 ```
 
+When running in the browser, WASM files need to be hosted and fetched before initializing. asm.js can be bundled together with the lib. In this case WASM should be faster.
+
+When running on the Node.js server, native should be faster. However there are Node.js environments where native modules are not supported (e.g. StackBlitz's WebContainers), or other JS runtimes that support WASM (e.g. Vercel's Edge Runtime, CloudFlare's worker, or Deno).
```
Comment (wong2) [inline README.md:340]: Cloudflare Workers

File: README.md [hunk 2]
```diff
@@ -335,6 +335,14 @@ init(yoga)
 await satori(...)
 ```
 
+When running in the browser, WASM files need to be hosted and fetched before initializing. asm.js can be bundled together with the lib. In this case WASM should be faster.
```
Comment (styfle) [inline README.md:338]: Not just the browser - you need to init wasm files for Node.js too if you choose to import `satori/wasm`

Comment (mrkldshv) [reply README.md]: Just updated. Should be fine now!

**Review by styfle — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -335,6 +335,14 @@ init(yoga)
 await satori(...)
 ```
 
+When running in the browser or in the Node.js environment, WASM files need to be hosted and fetched before initializing. asm.js can be bundled together with the lib. In this case WASM should be faster.
+
+When running on the Node.js server, native modules should be faster. However there are Node.js environments where native modules are not supported (e.g. StackBlitz's WebContainers), or other JS runtimes that support WASM (e.g. Vercel's Edge Runtime, Cloudflare Workers, or Deno).
```
Comment (yisibl) [inline README.md:340]: Deno already supports Native addon.  See: https://github.com/yisibl/resvg-js/pull/154
cc @shuding

**Pull request merged**